conda/environment.devenv.yml

```diff
@@ -95,4 +95,4 @@ dependencies:
 - xerces-c
 - zlib
 - pip:
-  - ptvsd
+  - debugpy
```

Comment:
seems to be available on conda so might not need to be on the pip section.
https://anaconda.org/search?q=debugpy

Reply:
so I should move it in the list above correct?